src/queue.c

```diff
@@ -7955,7 +7973,7 @@ _dispatch_context_cleanup(void *ctxt)
 #pragma mark -
 #pragma mark dispatch_init
 
-#if !DISPATCH_USE_COOPERATIVE_WORKQUEUE
+#if !DISPATCH_USE_INTERNAL_WORKQUEUE && !DISPATCH_USE_COOPERATIVE_WORKQUEUE
```

Comment:
Instead of doing this, can we fold `!INTERNAL_WORKQUEUE` into the definition of `DISPATCH_USE_COOPERATIVE_WORKQUEUE`?

Reply:
There are two negatives there which are disjoint, so I'm not sure how that would help here.